servicex/databinder_models.py

```diff
@@ -79,6 +82,18 @@ def validate_did_xor_file(cls, values):
             raise ValueError("Must specify one of Dataset, XRootDFiles, or RucioDID.")
         return values
 
+    @model_validator(mode="before")
+    @classmethod
+    def truncate_long_sample_name(cls, values):
+        """
+        Truncate sample name to 128 characters if exceed
+        Print warning message
+        """
+        if len(values["Name"]) > 128:
+            print(logger.warning(f"Truncating Sample name to 128 characters for {values['Name']}"))
```

Comment:
I don't think there should be a print here. `logger.warning` should output to stdout

Reply:
I don't understand why it's there :)